src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -101,6 +101,9 @@ def load_arguments(self, _):
         with self.argument_context('backup policy ' + command) as c:
             c.argument('name', policy_name_type, options_list=['--name', '-n'], help='Name of the backup policy. You can use the backup policy list command to get the name of a policy.')
 
+    with self.argument_context('backup policy list-associated-items') as c:
+        c.argument('backup_management_type', backup_management_type)
```

Comment:
Maybe you can add help message

Reply:
Help message for backup_management_type parameter? It's already abstracted here https://github.com/MabOneSdk/azure-cli/blob/f321b1dff4b869c9a65c99289a06380891e24f65/src/azure-cli/azure/cli/command_modules/backup/_params.py#L38